coordinator/gscoordinator/kubernetes_launcher.py

```diff
@@ -394,7 +544,23 @@ def create_interactive_instance(self, object_id: int, schema_path: str):
         )
         return process
 
+    def create_interactive_instance(self, object_id: int, schema_path: str):
+        pod_name_list, _, _ = self._allocate_interactive_engine()
+        if pod_name_list is None:
+            raise RuntimeError("Failed to allocate interactive engine")
+        hosts = ",".join(pod_name_list)
+
+        engine_type = "engine"
+        if self._mode == "lazy":
+            engine_type = "interactive"
+
+        return self._distribute_interactive_process(
+            hosts, object_id, schema_path, engine_type
+        )
+
     def close_interactive_instance(self, object_id):
+        pod_name_list, _, _ = self._allocate_interactive_engine()
```

Comment:
There could be multiple sets of interactive engines. I think current design doesn't suffice.

Reply:
This close interactive instance just got the last created instance.